Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7-bullseye
+
+COPY . .
+
+RUN pip install -U pip \
+    && pip install -r requirements.txt --no-cache-dir \
+    && cd dss_crf \
+    && python setup.py install \
+    && cd ..
+
+ENTRYPOINT ["python", "infer.py", "--path_to_pretrained_model=/200.pth", "--input_dir=/input", "--output_dir=/output"]
```

Comment:
Let's create a separate file `run.py` that will contain the default running logic so we can run docker without arguments. Please, [see](https://github.com/prime-slam/line-detection-association-dockers/blob/main/detectors/F-Clip/run.py). It also seems to me that it is worth creating an adapter file that will convert the input and output of the algorithm into a format that is convenient for us, such as, for example, [here](https://github.com/prime-slam/line-detection-association-dockers/blob/main/detectors/F-Clip/adapter.py)